web/src/components/Content.tsx

```diff
@@ -54,12 +55,17 @@ export const Content = (props: ContentProps): ReactElement => {
       return <HorizontalLine />;
     },
     blockquote: GreenBox,
```

Comment:
Not your fault, but this tripped me up that there's a component declared in this file called `GreenBox` that's different from the `greenBox` we've been talking about.

Reply:
it will get deleted once the clean up work is done